cmd/brimcap/ztests/analyze-delete-tmpdir.yaml

```
@@ -0,0 +1,14 @@
+script: |
+  tmpdir=$(brimcap analyze -printtmp -suricata=F alerts.pcap 2>&1 >/dev/null \
+    | zed query -f text 'type=="TempDir" | cut path' -)
+  [ -d $tmpdir ] && echo "TempDir $tmpdir still exists when it should have been deleted." \
+    && exit 1
```

Rather than adding `-printtmp`, how about setting `TMPDIR` and `TMP` to control where temporary directories are created? (If `brimcap` removes everything properly, `ls d` should produce no output.)
```suggestion
  mkdir d
  TMPDIR=$PWD/d TMP=$PWD/d brimcap analyze -suricata=F alerts.pcap >/dev/null 2>&1
  ls d
```

---

Awesome, but better!